content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
 1. Set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 2. Follow the steps to configure each library, if any, to generate OpenTelemetry-compatible instrumentation via `ActivitySource`
 
+The following example demonstrates how to instrument the `Hangfire` OpenTelemetry integrations with the Datadog .NET SDK:
+
+```js
```

Comment:
I think GitHub supports C# highlighting

https://github.com/github-linguist/linguist/blob/main/lib/linguist/languages.yml

Reply:
Good point. You can use `csharp`.